packages/grafana-ui/src/components/ErrorBoundary/ShowErrorWithStack.tsx

```diff
@@ -0,0 +1,20 @@
+import React, { FunctionComponent } from 'react';
+import { ErrorBoundaryApi } from './ErrorBoundary';
+
+export interface Props extends ErrorBoundaryApi {
+  className: string;
+  title: string;
+}
+
+export const ShowErrorWithStack: FunctionComponent<Props> = ({ error, errorInfo, className, title }) => (
+  <div className={className}>
+    <h2>{title}</h2>
+    <details style={{ whiteSpace: 'pre-wrap' }}>
+      {error && error.toString()}
+      <br />
+      {errorInfo && errorInfo.componentStack}
```

Comment:
nit: What about rendering this in <pre>/<code>  ?

Reply:
I wanted to keep this as it was, but I can have a look.

Reply:
WDYT @dprokop, don't know if that such a big difference 
**Before**
![image](https://user-images.githubusercontent.com/562238/68210067-08aa3480-ffd5-11e9-8ed6-f4f03623243f.png)

**After**
![image](https://user-images.githubusercontent.com/562238/68209889-a6e9ca80-ffd4-11e9-9587-17cd462a0198.png)

Reply:
I prefer the one with monospaced font